File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.cpp
class Solution {
public:
    int mirrorDistance(int n) {
        int r = 0;
        int m = n;
        while (m > 0) {
            r *= 10;
            r += m % 10;
            m /= 10;
        }
      
        return abs(n - r);
    }
};

File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.py
class Solution(object):
    
        
    def mirrorDistance(self, n):
        
        """
        :type n: int
        :rtype: int
        """
        ans = 0
        m = n
        x = 0
        while m:
             x *= 10
             x += m % 10
             m //=10 

        return abs(n-x) 


